packages/shared-state/files/etc/config/shared-state

```diff
@@ -0,0 +1,2 @@
+config shared-state 'options'
+	#list candidates 'http://10.13.0.2/cgi-bin/shared-state/'
```

Comment:
shouldn't this be uncommented, and in a file called shared-state.example or something like that?
this is in the right place but with a config that is broken? or at least it looks like... so better to provide a sane default, or no default and expect it not to be there at all.

Reply:
it is not broken, and empty list is the correct default

Reply:
good... thanks for the clarification @G10h4ck